config/locales/devise.en.yml

```
@@ -57,6 +57,8 @@ en:
   errors:
     messages:
       already_confirmed: "has already been confirmed. Sign in."
+      blank: "can’t be blank"
+      confirmation: "doesn’t match new password"
       confirmation_period_expired: "needs to be confirmed within %{period}. Request a new account."
       expired: "has expired. Request a new token."
```

Would changing this string be enough to resolve the original bug?

```suggestion
      expired: "Token has expired. Request a new token."
```

---

Yep this does solve this case (I suppose there may be others but since this fixes definitely existing ones that seems fine).